src/Components/ParkToilets/ParkToilets.js

```diff
@@ -62,18 +62,14 @@ class ParkToilets extends Component {
         if (!this.state.safeToilets.includes(foundToilet)) {
             this.setState({ safeToilets: [...this.state.safeToilets, foundToilet] })
         }
-        //have to do fetch call to see what's in there
-
-            // postToilet(foundToilet)
-            //     .then(toilet => console.log('line 61', toilet))
-            //     .catch(error => this.setState({ error: error }))
     }
 
     postAToilet = (newToilet) => {
         console.log('post line 74', this.state)
 
         if(!this.state.postedToilets.includes(newToilet)) {
             postToilet(newToilet)
+                //are these necessary for a post?
                 // .then(toilet => console.log('line 61', toilet))
                 // .catch(error => this.setState({ error: error }))
         }
```

Comment:
Hey Lexy! Your code looks great! 

-On the specific note of how to handle your "loading" image when the park toilets fetch is loading: Make the function that sets State for "isLoading" to false the callback function for the fetch call. That will make it wait until your fetch is all done before triggering.

-